integration/benchmark/cmd/ci/main.go

```diff
@@ -163,60 +167,93 @@ func main() {
 	if err != nil {
 		log.Fatal().Err(err).Msgf("unable to init loader")
 	}
+
+	// run load
 	lg.Start()
 
-	testStartTime := time.Now()
-	time.Sleep(loadCase.duration)
-	lg.Stop()
-	testEndTime := time.Now()
+	testRunDuration := time.NewTicker(loadCase.duration)
```

Comment:
Realized as I was porting this change over to manual/main.go that I had been conflating `time.Ticker` and `time.Timer` this should be swapped out with `time.Timer` since it is a one off event. Will be in my next commit.